<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1273</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BinaryT</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>График нормированной поправки по частоте</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="frequencycorrection" name="widget_freqcorr" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1273</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Порт</string>
    </property>
    <addaction name="action_openPort"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_tables">
   <property name="windowTitle">
    <string>Таблицы</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="minimumSize">
     <size>
      <width>284</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget_tables"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_SatState">
   <property name="windowTitle">
    <string>Созвездия</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Отрисовка спутников за горизонтом</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SatStateLabel" name="widget_gpsConstellation" native="true"/>
     </item>
     <item>
      <widget class="SatStateLabel" name="widget_glonassConstellation" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_openPort">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SatStateLabel</class>
   <extends>QWidget</extends>
   <header>satstatelabel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>frequencycorrection</class>
   <extends>QWidget</extends>
   <header>frequencycorrectionplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
